test/lifecycle_test.dart

```diff
@@ -545,6 +606,7 @@ void sharedLifecycleTests<T extends react.Component>({
       expect(
           component.lifecycleCalls,
           equals([
+            isComponent2 ? matchCall('getDerivedStateFromProps') : null,
```

Comment:
This should have `args: ` expectations to test that the correct arguments are being passed in